l2tdevtools/build_helpers/dpkg.py

```diff
@@ -525,7 +541,8 @@ def Build(self, source_helper_object):
     exit_code = subprocess.call('(cd {0:s} && {1:s})'.format(
         source_directory, command), shell=True)
     if exit_code != 0:
-      logging.error('Running: "{0:s}" failed.'.format(command))
+      logging.error('Running: "(cd {0:s} && {1:s}" failed.'.format(
```

Comment:
Maybe change to: `Failed to run ... with exit code: ...`

from the current wording it is not directly clear things failed.

Reply:
Done